/*
 *  Description:  cycle rates for player drivers (in hertz)
 *
 *  Copyright (C) 2009 Austin Robot Technology                    
 *
 *  License: Modified BSD Software License Agreement
 *
 *  $Id$
 */

#ifndef _HERTZ_H
#define _HERTZ_H

/** @file
    
    @brief Cycle rates for various ART vehicle nodes (in hertz).
 */

#define HERTZ_APPLANIX		20
#define HERTZ_BRAKE		20
#define HERTZ_OBSERVERS		10
#define HERTZ_IOADR		10
#define HERTZ_LASER_FUSION	10
#define HERTZ_LASER_NOISE	10
#define HERTZ_MAPLANES		5
#define HERTZ_NAVIGATOR		20
#define HERTZ_PILOT		20
#define HERTZ_PSEUDO_LASER	10
#define HERTZ_SICKFAST		10
#define HERTZ_STEERING		20
#define HERTZ_THROTTLE		20
#define HERTZ_VEHICLE_TF	5
#define HERTZ_VELODYNE_PROC	10

#endif // _HERTZ_H
